### PR TITLE
Physical item dl

### PR DESCRIPTION
**Pull request by davidpmccormick**
Relates to #6695

Using the `DescriptionList` to display the physical items.

image

### CODE REVIEW

File: catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import { FunctionComponent, ReactElement, useState, useEffect } from 'react';
+import { FunctionComponent, useState, useEffect } from 'react';
 import {
   PhysicalItem,
   ItemsWork,
   CatalogueApiError,
 } from '@weco/common/model/catalogue';
-import Table from '@weco/common/views/components/Table/Table';
+// import Table from '@weco/common/views/components/Table/Table';
```
Comment (gestchild) [inline catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx:7]: can we get rid of this properly

Comment (davidpmccormick) [reply catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx]: done

File: catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx [hunk 2]
```diff
@@ -27,44 +31,47 @@ function getFirstPhysicalLocation(item) {
...
-  const accessTerms = physicalLocation?.accessConditions[0]?.terms || ' ';
+    physicalLocation?.accessConditions?.[0]?.status?.label || '';
+  const accessTerms = physicalLocation?.accessConditions[0]?.terms || '';
   const locationId = physicalLocation?.locationType.id;
   const locationLabel = physicalLocation && getLocationLabel(physicalLocation);
   const locationShelfmark =
     physicalLocation && getLocationShelfmark(physicalLocation);
   const shelfmark = `${locationLabel ?? ''} ${locationShelfmark ?? ''}`;
 
   if (locationId !== 'on-order') {
```
Comment (gestchild) [inline catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx:57]: I think we should probably get rid of this and the related code from l139 now we're moving in a new direction with the layout.

Comment (gestchild) [reply catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx:57]: it was when we didn't want stuff on order to be part of the table: https://app.zeplin.io/project/5aab926b4b6efde01259e959/screen/6093a75840e0eb31b669c2f4

**Review by gestchild — APPROVED** [review]

**Pull request merged**